README.md

```diff
@@ -183,9 +199,11 @@ validation report. There is also a `check()` method that returns an Exception
 only with the first error found:
 
 ```php
+import Respect\Validation\Exceptions\ValidationExceptionInterface
+
 try {
     $usernameValidator->check('really messed up screen#name');
-} catch(\InvalidArgumentException $e) {
+} catch(ValidationExceptionInterface $e) {
```

Comment:
Since you must change this file again, do you mind to rename `$e` to `$exception`?